docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Set up Amazon QuickSight
+description: First steps to set up Amazon QuickSight
+template: concept-topic-template
+last_updated: Oct 24, 2024
+---
+
+This document describes how to set up Amazon QuickSight.
+
+1. [Install the Amazon QuickSight feature]().
+  At this stage, the **Analytics** section is displayed in the Back Office, but the functionality is not available.
+2. To request Amazon QuickSight analytics, contact your Customer Success Manager.
```

Comment:
```suggestion
2. To request Business Intelligence powered by Amazon QuickSight, contact your Customer Success Manager at https://now.spryker.com/contact-analytics.
```

Reply:
we need to use the official SKU name here "Business Intelligence powered by Amazon QuickSight"